website/docs/overview/query/analytical-query.md

```diff
@@ -434,10 +541,18 @@ Now we can use `ftest` as a source in any clause.
    [HOST]/api/db/query
 ```
 
+</TabItem>
+
+<TabItem value="graphql">
+
 ```graphql
 Not supported.
 ```
 
+</TabItem>
+
+<TabItem value="sparql">
+
 ```sparql
 // SPARQL Note: When using SPARQL, omit the `$` in front-of built-in sources. In addition, if you want to specify an `ISO-8601` formatted wall clock time, replace all the `:` with `;`. For example, `"SELECT ?s ?o WHERE {  ?s   fdb2019-03-14T20;59;36;097Z:person/handle  ?o.}`. 
```

Comment:
this renders on a single line in the browser, add in linebreaks so it renders on multiple lines